src/components/Header/Header.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import s from "./Header.module.css";
+
+const HeaderLink = "https://i.pinimg.com/originals/d9/12/09/d91209340bdc005936c46323a62caaff.png"
```

Comment:
Consider changing the case of "HeaderLink" to upper case like so "HEADER_LINK".
For your information:
1. "YOUR_NAME" - such naming indicates the constants (not the "const" variables that do not change in the CURRENT SCOPE, but actual constants that will not change for the ENTIRE project)
2. "YourName" - such naming indicates classes, constructor functions (everything that is used with "new YourName()) and components (like React components)
3. "yourName" - such naming is used for variables and regular functions.

Reply:
complite